browser/main/SideNav/index.js

```diff
@@ -38,6 +39,84 @@ class SideNav extends React.Component {
     router.push('/trashed')
   }
 
+  handleSwitchFoldersButtonClick (e) {
+    let { router } = this.context
+    router.push('/home')
+  }
+
+  handleSwitchTagsButtonClick (e) {
+    let { router } = this.context
+    router.push('/alltags')
+  }
+
+  SideNavComponent (isFolded, isHomeActive, isStarredActive, isTrashedActive, storageList) {
+    let { location, data } = this.props
+    let component
+    if (!location.pathname.match('/tags') && !location.pathname.match('/alltags')) {
+      component = (
+        <div>
+          <SideNavFilter
+            isFolded={isFolded}
+            isHomeActive={isHomeActive}
+            handleAllNotesButtonClick={(e) => this.handleHomeButtonClick(e)}
+            isStarredActive={isStarredActive}
+            isTrashedActive={isTrashedActive}
+            handleStarredButtonClick={(e) => this.handleStarredButtonClick(e)}
+            handleTrashedButtonClick={(e) => this.handleTrashedButtonClick(e)}
+          />
+
+          <div styleName='storageList'>
+            {storageList.length > 0 ? storageList : (
+              <div styleName='storageList-empty'>No storage mount.</div>
+            )}
+          </div>
+          <button styleName='navToggle'
+            onClick={(e) => this.handleToggleButtonClick(e)}
+          >
+            {isFolded
+              ? <i className='fa fa-angle-double-right' />
+              : <i className='fa fa-angle-double-left' />
+            }
+          </button>
+        </div>
+      )
+    } else {
+      component = (
+        <div>
+          <div styleName='tag-title'>
+            <p>Tags</p>
+          </div>
+          <div styleName='tagList'>
+            {this.tagListComponent(data)}
+          </div>
+        </div>
+      )
+    }
+
+    return component
+  }
+
+  tagListComponent (data) {
+    let tagList = data.tagNoteMap.map((tag, key) => {
+      return key
+    })
+    return (
+      tagList.map(tag => {
+        return (<TagListItem
```

Comment:
You don't need to write `return`
same as https://github.com/BoostIO/Boostnote/pull/922/files#r143006528